honeybadgermpc/hbavss.py

```diff
@@ -20,9 +20,12 @@
 
 class HbAVSSMessageType:
     OK = "OK"
+    IMPLICATE = "IMPLICATE"
```

Comment:
Please remove this, you don't seem to be using it anywhere.

Reply:
I'll use it later. I commented it out for now.